janno_details.md

```diff
@@ -45,108 +46,108 @@ The `Relation_Note` column allows to add free-text information about the relatio
+The column `Data_Preparation_Pipeline_URL` should finally store an URL that links to a complete and human-readable description of the computational pipeline (for example a specific configuration for nf-core/eager [@FellowsYates2021](https://doi.org/10.7717/peerj.10947)) by which the sample data was processed.
 
 #### Data yield
 
-The `Endogenous` column holds the percentage of mapped reads over the total amount of reads that went into the mapping pipeline. That boils down to the DNA percentage of the library that matches the (human) reference. It should be determined from Shotgun libraries (so before any hybridization capture), not on target and without any quality filtering. In case of multiple libraries only the highest value should be reported. The % endogenous DNA can be calculated for example with the [endorS.py](https://github.com/aidaanva/endorS.py) script.
+The `Endogenous` column holds the percentage of mapped reads over the total amount of reads that went into the mapping pipeline. That boils down to the DNA percentage of the library that matches the (human) reference. It should be determined from Shotgun libraries (so before any hybridization capture), not on target (i.e. across the whole genome, not specific positions), and before any mapping quality filtering. In case of multiple libraries only the highest value should be reported. The % endogenous DNA can be calculated for example with the [endorS.py](https://github.com/aidaanva/endorS.py) script.
 
 The `Nr_SNPs` column gives the number of SNPs reported in the genotype data files for this individual.
 
-The `Coverage_on_Target_SNPs` column reports the mean SNP coverage on the target SNP array (e.g. 1240K) for the merged libraries of this sample. To calculate the coverage it is necessary to determine which SNPs are covered how many times by the mapped reads. Individual SNPs might be covered multiple times, whereas others may not be covered at all by the highly deteriorated ancient DNA. The coverage for each SNP is therefore a number between 0 and n. The statistic can be determined for example with the QualiMap [@Okonechnikov2015](https://doi.org/10.1093/bioinformatics/btv566) software package. In case of multiple libraries, the coverage can be given as a mean across all of them.
+The `Coverage_on_Target_SNPs` column reports the mean fold coverage on the SNP set of the genotype dataset (e.g. 1240K) for the merged libraries of this sample. To calculate the coverage it is necessary to determine which SNPs are covered how many times by the mapped reads. Individual SNPs might be covered multiple times, whereas others may not be covered at all by the highly deteriorated ancient DNA. The coverage for each SNP is therefore a number between 0 and n. The statistic can be determined for example with the QualiMap [@Okonechnikov2015](https://doi.org/10.1093/bioinformatics/btv566) software package. In case of multiple libraries, the coverage can be given as a mean across all of them.
```

Comment:
This is not right. When compiling multiple coverages, the reported value should not be the mean, but probably the sum?
If I have a library with 2X 1240k coverage and another with 1X, merging them would give an average coverage of 3X, not 1.5X. 
[janno_columns.tsv](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv) might also need updating accordingly.

Reply:
The definition in the schema says

> average X-fold coverage across targeted SNP sites after quality filtering

I think this does not say anything about what should be done in case of multiple libraries, so we're safe there, as far as my understanding goes?

Now regarding the statement here:

> In case of multiple libraries, the coverage can be given as a mean across all of them.

I guess you're right. The mean doesn't make much sense here. It's rather the maximum or something between the maximum and the sum, right? I think we can just omit this sentence entirely.

Reply:
Well, I think multiple libraries are typically merged (on a stream or explicitly) when piped into a caller, so I agree that I think it should be the sum. This is somewhat awkward, though, if different subsets of libraries are subjected to different calling algorithms (as for example in-house with single- and double-stranded data). But even then, I think the sum of the coverages is somewhat what tells us something about the amount of data to expect. So yes, I think it should be the sum. I suggest:

```suggestion
The `Coverage_on_Target_SNPs` column reports the mean fold coverage on the SNP set of the genotype dataset (e.g. 1240K) for the merged libraries of this sample. To calculate the coverage it is necessary to determine which SNPs are covered how many times by the mapped reads. Individual SNPs might be covered multiple times, whereas others may not be covered at all by the highly deteriorated ancient DNA. The coverage for each SNP is therefore a number between 0 and n. The statistic can be determined for example with the QualiMap [@Okonechnikov2015](https://doi.org/10.1093/bioinformatics/btv566) software package. In case of multiple libraries, the total coverage should be given across all libraries.
```

Reply:
can we change 1240K to 1240k because reasons?

Reply:
1240K is the code that has been defined for this snp set across Poseidon, including in the POSEIDON.yml. I think consistency is more important than technical correctness. We're in too deep by now.

Reply:
Oh boy, so sorry, @wolfgangaroo. I think that ship has sunken already.